packages/components/image-viewer/__tests__/image-viewer.test.tsx

```diff
@@ -58,4 +58,19 @@ describe('<image-viewer />', () => {
     expect(imgList[1].attributes('style')).not.contains('display: none;')
     wrapper.unmount()
   })
+
+  test('image activeIndex total', async () => {
```

Comment:
```suggestion
  test('image progress render', async () => {
```
The description here could be tweaked a bit.